vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -599,6 +599,7 @@ GlobalOptions.sound_global_mute=Mute Module Sounds
 GlobalOptions.sound_wakeup_mute=Mute Server "Wake Up" Sounds
 GlobalOptions.old_continuation=Warn when using Legacy Load Continuation?
 GlobalOptions.send_to_location_movement_trails=Send To Location trait updates Movement Trails?
+GlobalOptions.stack_viewer_order=Reverse left-to-order of pieces in Stack Viewer
```

Comment:
Typo - Missing some words 'Reverse left-to-right order'?

Reply:
Fixed- ack!